.github/pull_request_template.md

```diff
@@ -1,9 +1,16 @@
 ## Issue
 
+<!-- If you want to close a issue with your PR use magic words such as closes, fixes or resolves "issue number" to autmatically link a PR to an issue. For example: Closes #0000, Fix: #0000 or Resolves: #0000 -->
+
 ## Description
 
 <!-- Explains the goal of this PR -->
 
 ### Preview
 
 <!-- Please add screenshots and/or gif that shows visual changes (if applicable) -->
+
+### Double check
+
+- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
+- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.
```

Comment:
In an ideal world this should be handled automatically by tools, but until then it's nice to add the check here :)

Reply:
Yeah I had a quick discussion with Pierre about it and the last one is already checked by the bots (but people seem to forget it anyway 😅). The first one is a lot trickier, we would have to calculate the changed parsers and then run them (and hope the source isn't down) or require tests to be implemented in each parser.

Reply:
The second one should ideally be a pre-commit thing so that forgetting it is not an option :D 

Good point about the first one!

Reply:
Let's remove the second one when we add the pre-commit 👍🏼